src/DynamoCoreWpf/Views/Preview/WatchTree.xaml

```diff
@@ -158,7 +160,45 @@
                                     <Setter Property="Visibility"
                                             TargetName="Expander"
                                             Value="Collapsed" />
+                                    <Setter Property="Margin"
+                                            TargetName="TreeViewItem"
+                                            Value="19,0,0,0" />
                                 </Trigger>
+
+                                <!-- __________________MultiDataTriggers______________________ -->
+                                <MultiDataTrigger>
+                                    <MultiDataTrigger.Conditions>
+                                        <!-- If treeview item is root item. -->
+                                        <Condition Binding="{Binding RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type TreeViewItem}}}"
+                                                   Value="{x:Null}" />
+                                        <!-- If root treeview item has child items. -->
+                                        <Condition Binding="{Binding RelativeSource={RelativeSource Self}, Path=HasItems}"
+                                                   Value="true" />
+                                    </MultiDataTrigger.Conditions>
+                                    <MultiDataTrigger.Setters>
+                                        <Setter Property="Margin"
+                                                TargetName="TreeViewItem"
+                                                Value="4,0,0,0" />
+                                    </MultiDataTrigger.Setters>
+                                </MultiDataTrigger>
+
+                                <MultiDataTrigger>
```

Comment:
This trigger is for such situation:
![image](https://cloud.githubusercontent.com/assets/8158404/11561078/7786352e-99cd-11e5-9e58-99e7f3d333dd.png)